launch.md

```diff
@@ -70,7 +70,7 @@ Set or change the following options to control VS Code's behavior during debuggi
 The following options enable you to modify the state of the target application when it is launched:
 
 * #### `args`
-  JSON array of command line arguments to pass to the program when it is launched. Example `["arg1", "arg2]`.
+  JSON array of command line arguments to pass to the program when it is launched. Example `["arg1", "arg2"]`. If you are escaping characters you will need to double escape them. For example `["{///"arg1///": true}]` will send `{"arg1": true}` to your application.
```

Comment:
Are these supposed to be backslashes?

Reply:
yes that is correct. I need to fix it.

Reply:
This isn't working. Double escaping is stil broken

Reply:
@vbaurea72 can you open an issue with more detail?